schedule/alp/alp_yast_as_workload.yaml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
it is already inside /alp/ folder, should be just `yast_as_workload.yaml`

Reply:
Done